include/crm/common/ipc_attrd.h

```diff
@@ -97,10 +105,14 @@ int pcmk__attrd_api_purge(pcmk_ipc_api_t *api, const char *host);
  *
  * \param[in] api           Connection to pacemaker-attrd
  * \param[in] host          Look up the attribute on this host
- *                          (or NULL for all hosts)
+ *                          for the current node, NULL for all nodes, otherwise
+ *                          the name of a single node)
```

Comment:
Leftover?

Reply:
With the other changes around calling pcmk__node_attr_target, this no longer exists.